CMakeLists.txt

```diff
@@ -342,7 +342,8 @@ macro(add_flang_library name)
   endif( LLVM_COMMON_DEPENDS )
 
   llvm_config( ${name} ${LLVM_LINK_COMPONENTS} )
-  target_link_libraries( ${name} ${LLVM_COMMON_LIBS} )
+  list(APPEND LLVM_COMMON_LIBS Dbghelp.lib)
```

Comment:
This should be included in windows only.

Reply:
Good catch.